src/crews.js

```diff
@@ -1,6 +1,8 @@
 const { execQuery } = require('./db');
 const moment = require('moment');
 
+const DEV = process.env.ENV == 'dev' ? true : false;
```

Comment:
This can be rewritten `const DEV = process.env.ENV == 'dev';` because that itself returns `true` or `false`.